experiments/azure/cyclecloud/cpu/results/128-nodes/lammps/lammps-128n-97-iter-1.out

```diff
@@ -0,0 +1,671 @@
+      bin: standard
+  (2) fix qeq/reax, perpetual, copy from (1)
+      attributes: half, newton off
+      pair build: copy
+      stencil: none
+      bin: none
+Setting up Verlet run ...
+  Unit style    : real
+  Current step  : 0
+  Time step     : 0.1
+Per MPI rank memory allocation (min/avg/max) = 252.6 | 252.8 | 253 Mbytes
+   Step          Temp          PotEng         Press          E_vdwl         E_coul         Volume    
+         0   300           -113.27833      439.02048     -111.57687     -1.7014647      4.4923072e+08
+        10   300.76708     -113.28045      807.30078     -111.57901     -1.7014362      4.4923072e+08
+        20   302.42351     -113.28524      1696.9908     -111.58392     -1.7013253      4.4923072e+08
+        30   302.57588     -113.28557      4297.9273     -111.58447     -1.7010962      4.4923072e+08
+        40   300.62223     -113.27967      6393.3851     -111.5789      -1.7007688      4.4923072e+08
+        50   297.4189      -113.27008      6471.1308     -111.56968     -1.7004027      4.4923072e+08
+        60   294.72953     -113.26202      6185.7111     -111.56199     -1.7000318      4.4923072e+08
+        70   294.6668      -113.26179      6831.2232     -111.56213     -1.6996547      4.4923072e+08
+        80   297.78807     -113.27107      8233.8672     -111.5718      -1.699276       4.4923072e+08
+        90   301.67958     -113.28267      9372.0856     -111.58373     -1.6989385      4.4923072e+08
+       100   302.56511     -113.28525      10370.591     -111.58656     -1.698695       4.4923072e+08
+Loop time of 56.2059 on 12288 procs for 100 steps with 39845888 atoms
+
+Performance: 0.015 ns/day, 1561.274 hours/ns, 1.779 timesteps/s, 70.893 Matom-step/s
+99.8% CPU use with 12288 MPI tasks x 1 OpenMP threads
+
+MPI task timing breakdown:
+Section |  min time  |  avg time  |  max time  |%varavg| %total
+---------------------------------------------------------------
+Pair    | 20.405     | 23.294     | 25.607     |  12.3 | 41.44
+Neigh   | 0.3577     | 0.37424    | 0.45794    |   1.0 |  0.67
+Comm    | 0.044294   | 1.3685     | 5.1089     |  63.1 |  2.43
+Output  | 0.55514    | 0.7776     | 0.89315    |   4.9 |  1.38
+Modify  | 29.366     | 30.389     | 31.201     |   7.0 | 54.07
+Other   |            | 0.003265   |            |       |  0.01
+
+Nlocal:        3242.67 ave        3269 max        3209 min
+Histogram: 23 549 2171 1246 116 425 2815 3833 1068 42
+Nghost:        12107.8 ave       12146 max       12063 min
+Histogram: 61 401 1227 1688 1299 1933 3079 1959 573 68
+Neighs:    1.07025e+06 ave 1.07789e+06 max 1.06022e+06 min
+Histogram: 40 602 2069 1230 171 520 2842 3728 1040 46
+
+Total # of neighbors = 1.3151174e+10
+Ave neighs/atom = 330.05097
+Neighbor list builds = 5
+Dangerous builds not checked
+Total wall time: 0:00:58
+[1725067507.397794] [cluster-256-hpc-256:82521:0]        mm_posix.c:291  UCX  ERROR shm_unlink(/ucx_shm_posix_2705a14c) failed: No such file or directory
```

Comment:
This one too. I think there are more in this directory (and I'll stop pointing them out) a grep would tell us how many.